src/lib.rs

```diff
@@ -2,24 +2,29 @@
 //!
 //! This crate provides a library for working with [Passage](https://passage.id), a modern passwordless authentication experience based on passkeys and magic links. This library currently implements a very small subset of the available Passage API endpoints.
 //!
-//! See Passage [Authentication API](https://docs.passage.id/api-docs/authentication-api) and [Management API](https://docs.passage.id/api-docs/management-api)  for all the possible endpoints that could be added to this crate in the future.
+//! See Passage [Authentication API](https://docs.passage.id/api-docs/authentication-api) and [Management API](https://docs.passage.id/api-docs/management-api) for all the possible endpoints that could be added to this crate in the future.
 //!
 //! ## Usage
 //!
-//! This crate is on [crates.io](https://crates.io/crates/passage-id) and can be used by adding `passage-id` to your dependencies in your project's Cargo.toml.
+//! This crate is published on [crates.io](https://crates.io/crates/passage-id) and can be added with `cargo add passage-id` or by manually adding `passage` to your Cargo.toml dependencies.
 //!
 //! ```toml
 //! [dependencies]
-//! passage-id = "latest"
+//! passage-id = "0.1"
 //! ```
 //!
 use jsonwebkey as jwk;
 use jsonwebtoken as jwt;
 use serde::Deserialize;
 
-/// Passage is the main entry point you'll be working with. Create with [Passage::new].
+/// Passage is the main entry point you'll be working with. See [Passage::new] for details on where to find these values for your app.
 pub struct Passage {
     app_id: String,
+    #[allow(dead_code)]
+    api_key: String,
+    #[allow(dead_code)]
+    base_uri: String,
+    pub_jwk: String,
```

Comment:
I know these aren't public, but maybe we should include doc comments just for the sake of other contributors?